src/App.vue

```diff
@@ -0,0 +1,29 @@
+<template>
+  <div id="app">
+    <Header />
+    <HeaderSwitch />
+    <router-view/>
+  </div>
+</template>
+
+<script>
+import Header from './components/Header.vue';
+import HeaderSwitch from './components/HeaderSwitch.vue';
+
+export default {
+  name: 'App',
+  components: {
+    Header,
+    HeaderSwitch,
+  },
+  data() {
+    return {
+      device: 'desktop',
```

Comment:
is this data used?

Reply:
Not anymore, components get this data from the store directly